src/User.java

```diff
@@ -0,0 +1,111 @@
+import java.util.Scanner;
+
+/**
+ * Created by melmo on 11/30/16.
+ */
+public class User {
+    private String name;
+    private int balance;
+    private Scanner input = new Scanner(System.in);
+
+    public User(int startBalance) {
+        this.balance = startBalance;
+    }
+
+    public String getName() {
+        return this.name;
+    }
+
+    public int getBalance() {
+        return this.balance;
+    }
+
+    public void chooseName() throws Exception {
+        System.out.println("What is your name?");
+        String name = this.input.nextLine();
+        if (!name.equals("")) {
```

Comment:
I would use `name.isEmpty()`.  It does the same thing but is more descriptive and easier to read.